packages/cli/src/index.ts

```diff
@@ -1,9 +1,24 @@
+import { existsSync } from "fs-extra";
 import "dotenv/config";
 import yargs from "yargs/yargs";
 import * as preview from "./commands/preview";
 import * as init from "./commands/init";
+import { looksLikeTypescriptProject, MAILING_CONFIG_FILE } from "./config";
 
+const fs = require("fs");
+
+const config = existsSync(MAILING_CONFIG_FILE)
+  ? JSON.parse(fs.readFileSync(MAILING_CONFIG_FILE))
+  : {};
+
+// prettier-ignore
 yargs(process.argv.slice(2))
+  .boolean(['quiet', 'typescript'])
+  .config(config)
+  .default('port', 3883)
```

Comment:
Can this use the `DEFAULT_PORT` constant from `packages/cli/src/commands/preview.ts`?

Reply:
This should actually supercede DEFAULT_PORT, but still allow you to pass a --port option to preview and pick it up correctly